paasta_tools/chronos_tools.py

```diff
@@ -597,6 +596,27 @@ def get_key(job):
     )
 
 
+def get_chronos_status_for_job(client, service, instance):
+    """
+    Returns the status (queued, idle, running, etc.) for a specific job from
+    the undocumented Chronos csv api that has the following format:
+
+    node,example_job,success,queued
+    node,parent_job,success,idle
+    node,child_job,success,idle
+    link,parent_job,child_job
+
+    :param client: Chronos client object
+    :param service: service name
+    :param instance: instance name
+    """
+    resp = client._call("/scheduler/graph/csv", "GET")
```

Comment:
The endpoint returns immediately, but the actual state it returns occasionally takes a second to be updated (killing a job that was "queued" is really the only the case where this happens) which is why the tests call this repeatedly.